Rakefile

```diff
@@ -15,7 +15,7 @@ end
 RuboCop::RakeTask.new
 
 RSpec::Core::RakeTask.new(:spec) do |r|
-  r.pattern = FileList['**/**/*_spec.rb']
+  r.pattern = FileList['spec/*-spec.rb']
```

Comment:
any reason to not just add the another path rather than removing?

Reply:
or just name the specs `whatever_spec.rb` to follow the convention used in all the plugins.

Reply:
:+1:

Reply:
I have reverted this change.

The names of scripts under bin directory have dashes. They are not following [RubyGems Guides](http://guides.rubygems.org/name-your-gem/) which require names to use underscores for multiple words and use dashes for extensions. Do names of scripts need to be changed to use underscores instead of dashes (e.g. `check-all-gpg-keys-for-expiry.rb` becomes `check_all_gpg_keys_for_expiry.rb`)?